README.md

```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## My Brief Biography
+
+My name is **Mercedes Gonzalez**.
+
+I am Mexican and have lived in Mexico City all my life.
+
+![Mexico](https://es.wikipedia.org/wiki/M%C3%A9xico#/media/Archivo:Diplomatic_missions_of_Mexico4.png)
```

Comment:
Nice picture! You can also use relative paths to link pictures that you have on your laptop.